assets/css/frontend/style.css

```diff
@@ -18,4 +18,4 @@
 @import url("templates/index");
 
 /* Components */
-@import url("components/index");
+@import url("components/index");
```

Comment:
Minor thing; it looks like this file got edited and lost the final newline, not respecting this .editorconfig rule:

https://github.com/10up/theme-scaffold/blob/master/.editorconfig#L6

Reply:
Bleh, I'll update that another day. Thanks for the note though!

Reply:
I’ve got you covered. 😉  https://github.com/10up/theme-scaffold/pull/32

Reply:
thanks homie